lib/rbac/filterer.rb

```diff
@@ -228,6 +232,10 @@ def filtered(objects, options = {})
       Rbac.search(options.reverse_merge(:targets => objects)).first
     end
 
+    def filter(object, options = {})
+      Rbac.search(options.reverse_merge(:targets => [object])).first.first
```

Comment:
Is this being exposed so we can optimize it later?  As you say, it's no less worse than the existing implementation but changes the location of the `.first` call.